packit_service/worker/handlers/forges.py

```diff
@@ -90,3 +96,119 @@ def run(self) -> TaskResults:
+        if (
+            self.service_config.deployment == Deployment.prod
+            and issue_author != "packit-as-a-service[bot]"
+        ) or (
+            self.service_config.deployment == Deployment.stg
+            and issue_author != "packit-as-a-service-stg[bot]"
+        ):
+            logger.debug(
+                f"Packit verification comment command placed on issue with author "
+                f"other than our app: {issue_author}"
+            )
+            return False
+
+        return True
+
+    def run(self) -> TaskResults:
+        """
+        Discover information about organization/user which wants to verify the FAS account.
+        Allowlist automatically if mapping from github username to FAS account can prove a match.
+
+        Returns:
+            TaskResults
+        """
+        logger.debug(
+            f"Going to run verification of FAS account triggered by comment:"
+            f" {self.comment}"
+        )
+        # e.g. User Bebaabeni needs to be approved.
+        account_type, account_login, _ = self.issue.title.split(maxsplit=2)
+        namespace = f"github.com/{account_login}"
+        command_parts = get_packit_commands_from_comment(
+            self.comment, self.service_config.comment_command_prefix
+        )
+        # we expect ["verify-fas", "fas-account"]
+        if len(command_parts) != 2:
+            msg = "Incorrect format of the Packit verification comment command."
+            logger.debug(msg)
+            self.issue.comment(
+                f"{msg} The expected format: `/packit verify-fas my-fas-account`"
+            )
+            return TaskResults(success=False, details={"msg": msg})
+
+        fas_account = command_parts[1]
+
+        allowlist = Allowlist(service_config=self.service_config)
+        if allowlist.is_approved(namespace):
+            msg = (
+                f"Invalid namespace! {account_type} `{account_login}` already approved."
+            )
+            logger.debug(msg)
+            self.issue.comment(msg)
+            return TaskResults(success=False, details={"msg": msg})
+
+        if allowlist.verify_fas(
+            namespace=namespace, sender_login=self.sender_login, fas_account=fas_account
+        ):
+            msg = (
+                f"{account_type} `{account_login}` approved successfully "
+                f"using FAS account `{fas_account}`!"
+            )
+            logger.debug(msg)
+            self.issue.comment(msg)
+            self.issue.close()
+
+        else:
+            msg = (
+                f"We were not able to find a match between GitHub Username field in FAS account"
+                f" `{fas_account}` and Github user `{self.sender_login}`. Please, check that you "
+                f"have set the field correctly and try again or contact "
+                f"[us](https://packit.dev/#contact)."
+            )
+            logger.debug(msg)
+            self.issue.comment(msg)
```

Comment:
What about giving :+1: / :-1: to the original comment?  (Definitely not needed..;)

Reply:
I wanted to be consistent with the behaviour for the other Packit comment commands, so implemented only :eyes: (https://github.com/packit/packit-service/blob/e5897986314acf955944829385898947070ca62f/packit_service/worker/jobs.py#L576). Although thinking about it now, I am not sure whether that won't be confusing for newcomers who have never used Packit before :D

Reply:
I think that's okay. It means, that Packit noticed the command and will do something about it. It's also a good indication for us and can help with debugging issues.